server/src/main.rs

```diff
@@ -75,7 +78,15 @@ fn main() {
                     })
             })
         });
+
+    let dump_content = warp::path("dump_content")
+        .map(|| {
+            let suspect_backgrounds = &*content::suspect_backgrounds;
```

Comment:
the `&*` should be redundant, but I'm guessing it's here to fix an error?

Reply:
Hmm yeah, it's my understanding that `lazy_static` creates a special type to represent the wrapped value, which then needs to be de-referenced to access the actual vector (in this case). There may be a better way to do that? This certainly doesn't work without the `&*`:

<img width="842" alt="screen shot 2018-10-15 at 11 03 56 pm" src="https://user-images.githubusercontent.com/1100824/46944457-99faf000-d0ce-11e8-9ca7-7c97ef82a7e1.png">